lib/tests/run_tests.js

```diff
@@ -140,9 +140,14 @@ module.exports = {
               console.log(`Coverage report created. You can find it here: ${fs.dappPath('coverage/__root__/index.html')}\n`);
               const opn = require('opn');
               const _next = () => { next(); };
-              opn(fs.dappPath('coverage/__root__/index.html'), {wait: false})
-                .then(() => new Promise(resolve => setTimeout(resolve, 1000)))
-                .then(_next, _next);
+              if (!options.noBrowser) {
+                opn(fs.dappPath('coverage/__root__/index.html'), {wait: false})
```

Comment:
Could you do an early return here so that we can remove the `else`. This is just an aesthetic change, but preferred.